habushu-maven-plugin/src/test/java/org/technologybrewery/habushu/ReportFormattingSteps.java

```diff
@@ -0,0 +1,16 @@
+package org.technologybrewery.habushu;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class ReportFormattingSteps {
+    @When("the Habushu project is built")
+    public void the_habushu_project_is_built() {
+        // Manual
+    }
+
+    @Then("a Cucumber report file {string} exists")
+    public void a_cucumber_report_file_exists(String string) {
+        // Manual
```

Comment:
what is meant by just the manual comment? some more context in the comment would help with knowing why these steps are empty

Reply:
`public class ReportFormattingSteps {
    @When("the Habushu project is built")
    public void the_habushu_project_is_built() {
        // Manual - test is dependent on files generated after Behave runs and cannot be automated
    }

    @Then("a Cucumber report file {string} exists")
    public void a_cucumber_report_file_exists(String string) {
        // Manual - test is dependent on files generated after Behave runs and cannot be automated
    }
`